cmd/kata-local-ci

```diff
@@ -0,0 +1,340 @@
+#!/bin/bash
+#
+## Usage: kata-local-ci <options>
+##   Run or setup local CI environment for kata containers using libvirt.
+##
+## Options include:
+##   -d <distro>: Select VM distro (fedora,ubuntu,centos, default:fedora)
+##   -f: force install of the VM (and overwrite disk)
+##   -F: "Faster" by reusing the image without reverting to earlier snapshot
+##   -h: Print help
+##   -i: Install required packages
+##   -j <job>: CI job to run (default: all)
+##   -r <option>: Pass extra option to rsync (e.g. exclude dirs)
+##   -w <dir>: Work directory to test (default $GOPATH/src/github.com/kata-containers)
+##
+## Creation options include:
+##   -c <cpus>: Number of CPUs for the VM (default 4)
+##   -k <ssh_key_file> (default ~/.ssh/id_rsa.pub)
+##   -m <memory>: Memory for the VM in megabytes (default 8192M)
+##   -n network config (optional)
+##   -o <disk-image>: Path and image name for the VM
+##   -s <size>: Disk image size for the VM (default 50G)
+##
+# Remotely inspired by https://github.com/alicefr/kata-local-ci-setup
+# by Alice Frosi (@alicefr), but using a different single-script
+# approach and with a focus on being able to suspend/resume
+#
+
+GOPATH=~/go
+distro=fedora
+size=50G
+memory=8192
+cpus=4
+key=~/.ssh/id_rsa.pub
+vmdir=go/src/github.com/kata-containers
+directory=$GOPATH/src/github.com/kata-containers
+disk=""
+network=""
+job="CRIO_K8S"
+rsync_options="--exclude=kata-containers/src/agent/target"
+kci=~/.kata-containers-ci
+install=false
+revert=true
+ubuntu20_workaround=false
```

Comment:
Does the recently released Ubuntu 22.04 LTS also need this workaround?

Reply:
Yes, and not having to workaround in this ugly way is one of the reasons this whole mess has not landed yet.